### ISSUE TITLE
Already running instance of headless Chrome makes tests fail

### ISSUE THREAD

**Issue by danrr** [NimaSoroush/differencify#4]
I'm getting test failures with

 ```console.error node_modules/differencify/dist/logger.js:53
      Error: Failed to launch a browser.
          at Chromy.start$ (/node_modules/chromy/dist/index.js:147:21)
          at tryCatch (/node_modules/regenerator-runtime/runtime.js:65:40)
          at Generator.invoke [as _invoke] (/node_modules/regenerator-runtime/runtime.js:303:22)
          at Generator.prototype.(anonymous function) [as next] (/node_modules/regenerator-runtime/runtime.js:117:21)
          at tryCatch (/node_modules/regenerator-runtime/runtime.js:65:40)
          at invoke (/node_modules/regenerator-runtime/runtime.js:155:20)
          at /node_modules/regenerator-runtime/runtime.js:165:13
```

whenever I run the tests with an already running instance of headless Chrome. Killing that instance fixes the test.

Adding checks for free ports and fixing #9 will fix this.

**Comment by NimaSoroush** [comment]
should be closed by #12

**Comment by danrr** [comment]
This isn't fully fixed by #12, if a port is being used by something else, we'll still fail the test.

**Comment by NimaSoroush** [comment]
oops. closed the wrong one!

**Comment by NimaSoroush** [comment]
should be closed by #25
